system-test/spanner.ts

```diff
@@ -1702,8 +1663,8 @@ describe('Spanner', () => {
         err => {
           assert.ifError(err);
 
-          database.run(
-            'SELECT * FROM Singers ORDER BY SingerId',
+          DATABASE.run(
+            'SELECT * FROM SingersTables ORDER BY SingerId',
```

Comment:
Could we create a "TABLE_NAME" for this describe block too (and any others that work off of a unique table)? Now that the Database instance is being re-used and hosting multiple tables, it would be an easy mistake for someone to add a new test in a particular describe block and query against the wrong table.